File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03042/p03042_num4_parsed.py
S = input()

first_part = int(S[:2])
second_part = int(S[2:])

if 1 <= first_part <= 12 and 1 <= second_part <= 12:
    print("AMBIGUOUS")
elif 1 <= first_part <= 12:
    print("MMYY")
elif 1 <= second_part <= 12:
    print("YYMM")
else:
    print("NA")

File: Datasets/TrickyBugs/GenProgs/tc_generated_progs_cpp/p03042/p03042_num4_parsed.cpp
#include <iostream> 
using namespace std;
int main()
{
    string s; 
    cin >> s;
    
    int a = stoi(s.substr(0, 2));
    int b = stoi(s.substr(2, 2));
    
    if(a > 0 && a < 13 && b > 0 && b < 13)
        cout << "AMBIGUOUS";
    else if((a == 0 || a > 12) && (b == 0 || b > 12))
        cout << "NA";
    else if(a > 0 && a < 13)
        cout << "MMYY";
    else
        cout << "YYMM";
}
